stdlib/SparseArrays/test/sparse.jl

```diff
@@ -3173,11 +3173,15 @@ end
 end
 
 @testset "issue #41135" begin
-    @test repr(SparseMatrixCSC([1;;])) == "[1;;]"
-    @test repr(SparseMatrixCSC([1 0; 0 2])) == "[1 0; 0 2]"
-    @test summary(SparseMatrixCSC([1;;])) == "1×1 SparseMatrixCSC{$Int, $Int} with 1 stored entry"
+    @test repr(SparseMatrixCSC([7;;])) == "sparse([1], [1], [7])"
+
+    m = SparseMatrixCSC([0 3; 4 0])
+    @test repr(m) == "sparse([2, 1], [1, 2], [4, 3])"
+    @test eval(Meta.parse(repr(m))) == m
+    @test summary(m) == "2×2 $SparseMatrixCSC{$Int, $Int} with 2 stored entries"
+
     m = sprand(100, 100, .1)
-    @test repr(m) == join(split(repr(MIME("text/plain"), m), '\n')[2:end], '\n')
+    @test occursin(r"^sparse\(\[.+\], \[.+\], \[.+\]\)$", repr(m))
```

Comment:
Maybe test `eval(Meta.parse(repr(m))) == m`

I would also test for something like `S = sparse([85, 5, 38, 37, 59], [19, 72, 76, 98, 162], [0.8, 0.3, 0.2, 0.1, 0.5], 100, 200)` rather than a random array.  This ensures that it will catch a bug like the missing `m,n` noted above, and you can also test that the output matches a specific string.

Reply:
Makes sense. I added that as another test case.